src/container/use-sticky-header.ts

```diff
@@ -92,8 +95,3 @@ export const useStickyHeader = (
     stickyStyles,
   };
 };
-
-export function useSupportsStickyHeader() {
-  const isMobile = useMobile();
-  return supportsStickyPosition() && !isMobile;
```

Comment:
Why we do not need to take `supportsStickyPosition()` into consideration? What about the server-side?

Reply:
Great call - I've actually overseen the SSR check. I've re-added it.